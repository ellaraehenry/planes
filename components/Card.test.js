// import React from 'react';
// import renderer from 'react-test-renderer';
// import Card from './Card';

// describe('Card', () => {
//   it('renders correctly', () => {
//     const tree = renderer
//       .create(<Card />)
//       .toJSON();
//     expect(tree).toMatchSnapshot();
//   });
  
//   it('renders name and img correctly', () => {
//     const name = 'name'
//     const img = require('../assets/images/logo.png')

//     const tree = renderer
//       .create(<Card name={name} img={img} />)
//       .toJSON();
//     expect(tree).toMatchSnapshot();
//   });
// });