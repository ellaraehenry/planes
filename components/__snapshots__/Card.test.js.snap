// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderRadius": 5,
      "elevation": 5,
      "flex": 1,
      "justifyContent": "center",
      "margin": 8,
      "paddingHorizontal": 20,
      "paddingTop": 20,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.15,
      "shadowRadius": 2,
    }
  }
>
  <Image
    style={
      Object {
        "borderRadius": 2,
        "flex": 1,
        "height": undefined,
        "resizeMode": "cover",
        "width": "100%",
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#4a4a4a",
        "fontSize": 25,
        "padding": 20,
      }
    }
  />
</View>
`;

exports[`Card renders name and img correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "borderRadius": 5,
      "elevation": 5,
      "flex": 1,
      "justifyContent": "center",
      "margin": 8,
      "paddingHorizontal": 20,
      "paddingTop": 20,
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 1,
        "width": 0,
      },
      "shadowOpacity": 0.15,
      "shadowRadius": 2,
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "borderRadius": 2,
        "flex": 1,
        "height": undefined,
        "resizeMode": "cover",
        "width": "100%",
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#4a4a4a",
        "fontSize": 25,
        "padding": 20,
      }
    }
  >
    name
  </Text>
</View>
`;
